### ISSUE TITLE
NERmodel fine tuning with mine labels/entities?

### ISSUE THREAD

**Issue by matteopulega** [ThilinaRajapakse/simpletransformers#461]
Hi, is there any way to simply fine tuning the NERmodel with my list of sentences and personal entities (COLOR, MATERIAL for example)? 

I tried to do

```
    model = NERModel(model_type='bert',  model_name='bert-base-multilingual-cased',labels=labels, use_cuda=False, args={'output_dir': "outputs_multilingual/"})

    # Train the model
    model.train_model(train_df)

    # Evaluate the model
    result, model_outputs, predictions = model.eval_model(eval_df)
```
but i don't know if it's the correct method to fine tuning

train_df and eval_df are dataframes with the suggested schema from simple-trainsformers.

**Comment by ThilinaRajapakse** [comment]
Yeah, you can use your [own labels](https://simpletransformers.ai/docs/ner-specifics/#custom-labels).

**Comment by matteopulega** [comment]
Thanks, this website is very useful! 
But one question: 
in https://simpletransformers.ai/docs/usage/
fine tuning is present only in "Language model training" model, but not in "named entity recognition" model. 
So, my question is: for NERmodel, if my vocabulary is particular compared to those used in the pre-trained models, can I teach this vocabulary to my model simply by training and passing my personal labels?
I don't understand if fine-tuning is necessary in some way or it would be enough to have a full-bodied training dataset and use the fiew above rows in previous message.

**Comment by ThilinaRajapakse** [comment]
You can try using a pretrained model for NER first but if the performance is lacking, you can try to fine-tune the pretrained model (language model fine-tuning) on your text before doing the NER.

However, that is still not going to add new tokens to the vocabulary. Generally speaking, adding new tokens is unnecessary. For the vast majority of cases, training a language model from scratch is not feasible and unlikely to yield better results than either using a pre-trained model or fine-tuning a pre-trained model.

**Comment by matteopulega** [comment]
Thank you, i find the fine-tuning interesting. The results are not enough acceptable. My problem is that i haven't a dataset very big. So, my question is:
is it better gave more data in fine-tuning step (language model fine-tuning) or in training step of the NER model?

**Comment by ThilinaRajapakse** [comment]
You can use the same training data for both the LM fine-tuning and the NER training. I don't think it will be an issue.
